### PR TITLE
:sparkles: Added GetOpenFileNames()

### PR DESCRIPTION
**Pull request by xRadne**
Added public method GetOpenFileNames() and check open files on RunAnalysis()

### CODE REVIEW

**Review by andosca — CHANGES_REQUESTED** [review]

File: src/Calculate/Application.cs [hunk 1]
```diff
@@ -149,6 +158,22 @@ public bool RunFdScript(FdScript fdScript, bool killProcess, bool endSession)
                 this.KillProcesses();
             }
```
[the remark this answers is not in the document]
Comment (xRadne) [reply src/Calculate/Application.cs:160]: Implemented!
👷‍♂️🍻👷‍♂️

File: src/Calculate/Application.cs [hunk 2]
```diff
@@ -147,7 +147,7 @@ public void OpenStruxml(string struxmlPath, bool killProcess)
         /// <param name="killProcess"></param>
         /// <param name="endSession"></param>
         /// <returns></returns>
-        public bool RunFdScript(FdScript fdScript, bool killProcess, bool endSession)
+        public bool RunFdScript(FdScript fdScript, bool killProcess, bool endSession, bool checkOpenFiles = true)
```
Comment (andosca) [inline src/Calculate/Application.cs:150]: Remove default argument.

**Review by andosca — COMMENTED** [review]
Set checkOpenFiles to false here.

https://github.com/strusoft/femdesign-api/blob/aeced73f2f7ea57f46dd741a1356f8541ded00b5/componentsGrasshopper/Model/ModelReadStr.cs#L55

https://github.com/strusoft/femdesign-api/blob/aeced73f2f7ea57f46dd741a1356f8541ded00b5/src/Model/Model.cs#L2394

File: src/Calculate/Application.cs (hunk 2 above)
Comment (xRadne) [reply src/Calculate/Application.cs:150]: Will keep the default argument

**Pull request merged**